frontend/src/metabase/browse/components/MetricsTable.tsx

```diff
@@ -0,0 +1,286 @@
+  const collectionWidth = 38.5;
+  const descriptionWidth = 100 - collectionWidth;
+
+  return (
+    <Table aria-label={skeleton ? undefined : t`Table of metrics`}>
+      <colgroup>
+        {/* <col> for Name column */}
+        <ModelNameColumn {...nameProps} />
+
+        {/* <col> for Collection column */}
+        <TableColumn {...collectionProps} width={`${collectionWidth}%`} />
+
+        {/* <col> for Description column */}
+        <TableColumn {...descriptionProps} width={`${descriptionWidth}%`} />
+
+        <Columns.RightEdge.Col />
+      </colgroup>
+      <thead>
+        <tr>
+          <SortableColumnHeader
+            name="name"
+            sortingOptions={sortingOptions}
+            onSortingOptionsChange={handleSortingOptionsChange}
+            {...nameProps}
+            style={{ paddingInlineStart: ".625rem" }}
+            columnHeaderProps={{
+              style: { paddingInlineEnd: ".5rem" },
+            }}
+          >
+            {t`Name`}
+          </SortableColumnHeader>
+          <SortableColumnHeader
+            name="collection"
+            sortingOptions={sortingOptions}
+            onSortingOptionsChange={handleSortingOptionsChange}
+            {...collectionProps}
+            columnHeaderProps={{
+              style: {
+                paddingInline: ".5rem",
+              },
+            }}
+          >
+            {t`Collection`}
+          </SortableColumnHeader>
+          <SortableColumnHeader
+            name="description"
+            sortingOptions={sortingOptions}
+            onSortingOptionsChange={handleSortingOptionsChange}
+            {...descriptionProps}
+            columnHeaderProps={{
+              style: {
+                paddingInline: ".5rem",
+              },
+            }}
+          >
+            {t`Description`}
+          </SortableColumnHeader>
+          <Columns.RightEdge.Header />
+        </tr>
+      </thead>
+      <TBody>
+        {skeleton ? (
+          <Repeat times={7}>
+            <MetricRow />
+          </Repeat>
+        ) : (
+          sortedMetrics.map((metric: MetricResult) => (
+            <MetricRow metric={metric} key={`${metric.model}-${metric.id}`} />
+          ))
+        )}
+      </TBody>
+    </Table>
+  );
+}
+
+function MetricRow({ metric }: { metric?: MetricResult }) {
+  const dispatch = useDispatch();
+
+  const handleClick = useCallback(
+    (event: React.MouseEvent) => {
+      if (!metric) {
+        return;
+      }
+
+      // do not trigger click when selecting text
+      const selection = document.getSelection();
+      if (selection?.type === "Range") {
+        event.stopPropagation();
+        return;
+      }
+
+      const { id, name } = metric;
+      const url = Urls.metric({ id, name });
+      const subpathSafeUrl = Urls.getSubpathSafeUrl(url);
+
+      // TODO:
+      // trackMetricClick(metric.id);
+
+      if ((event.ctrlKey || event.metaKey) && event.button === 0) {
```

Comment:
Please add tests for this.

Reply:
Cypress does not support handling browser tabs so I'm not sure how we would write a test for this. [My attempt at this] works locally but fails on CI.

Reply:
@nemanjaglumac has recently added similar tests:
- https://github.com/metabase/metabase/pull/46228/files#diff-9be7ff8d54b6281f22a862eb90f6318bd7104e920523e02b15e50345d55e84e7R34-R41
- https://github.com/metabase/metabase/pull/46228/files#diff-9be7ff8d54b6281f22a862eb90f6318bd7104e920523e02b15e50345d55e84e7R22-R27

Reply:
Yep just found them!